src/app/product-list-page/components/product-list-page/faceted-navigation/faceted-navigation.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { IFilter } from 'src/app/shared/interfaces/product';
+
+@Component({
+  selector: 'app-faceted-navigation',
+  templateUrl: './faceted-navigation.component.html',
+  styleUrls: ['./faceted-navigation.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class FacetedNavigationComponent {
+  filters: IFilter[] = [
+    {
+      'type': 'radio',
+      'name': 'gender',
+      'fields': ['man', 'woman', 'children']
+    },
+    {
+      'type': 'checkbox',
+      'name': 'category',
+      'fields': ['coats', 'panties', 'shoes', 'underwear']
+    },
+    {
+      'type': 'checkbox',
+      'name': 'size',
+      'fields': ['small', 'medium', 'large', 'x large']
```

Comment:
sizes should enumerable 

export enum ProductSize {
  S = 's',
  M = 'm',
  L = 'l',
  XL = 'xl'
}